scripts/scr_planetary_feature/scr_planetary_feature.gml

```diff
@@ -1,7 +1,264 @@
+	var awake_tomb = 0;
+	 var tombs = search_planet_features(planet, search_feature);
+	 if (array_length(tombs)>0){
+		 for (var tomb =0;tomb<array_length(tombs);tomb++;){
+			 if (planet[tombs[tomb]].awake == 1){
+				awake_tomb = 1;
+				planet[tombs[tomb]].awake = 0;
+				planet[tombs[tomb]].sealed = 1;
+			 }
+			 if (awake_tomb = 1){break;}
+		 }
+	 }
+}
+
+
+//awakens a tomb world so necrons and necron fleets will spawn
+function awaken_tomb_world(planet){
+	var awake_tomb = 0;
+	 var tombs = search_planet_features(planet, search_feature);
+	 if (array_length(tombs)>0){
+		 for (var tomb =0;tomb<array_length(tombs);tomb++;){
+			 if (planet[tombs[tomb]].awake == 0){
+				awake_tomb = 1;
+				planet[tombs[tomb]].awake = 1;
+			 }
+			 if (awake_tomb = 1){break;}
+		 }
+		 
+	 }
+}
+
+
+// creates alerts for discovering features on a planet
+function scr_planetary_feature(planet_num) {
+	var plan_feat_count = array_length(p_feature[planet_num]);
+	//need to iterate over features instead of just looking at first
+	for (var f= 0; f < plan_feat_count;f++;){
+		var feat = p_feature[planet_num][f];
+		if (feat.player_hidden ==1){
+			feat.player_hidden =0;
+			switch (feat.f_type){
+				case P_features.Sororitas_Cathedral:
+					if (obj_controller.known[5]=0) then obj_controller.known[5]=1;
+				    var lop;lop="Sororitas Cathedral discovered on "+string(name)+" "+scr_roman(planet_num)+".";debugl(lop);
+				    scr_alert("green","feature",lop,x,y);scr_event_log("",lop);
+				    if (p_heresy[planet_num]>10) then p_heresy[planet_num]-=10;
+				    p_sisters[planet_num]=choose(2,2,3);goo=1;
+					break;
+				case P_features.Necron_Tomb:
+				    var lop;lop="Necron Tomb discovered on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+				    scr_alert("red","feature",lop,x,y);scr_event_log("red",lop);
+					break;
+				case P_features.Artifact:
+					var lop;lop="Artifact discovered on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+					scr_alert("green","feature",lop,x,y);scr_event_log("",lop);
+					break;
+				case P_features.STC_Fragment:
+					var lop;lop="STC Fragment located on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+					 scr_alert("green","feature",lop,x,y);scr_event_log("",lop);
+					 break;
+				case P_features.Ancient_Ruins:
+					var lop;lop="Ancient Ruins discovered on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+					scr_alert("green","feature",lop,x,y);scr_event_log("",lop);
+					break;
+				case P_features.Cave_Network:
+					var lop;lop="Extensive Cave Network discovered on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+			        scr_alert("green","feature",lop,x,y);scr_event_log("",lop);
+					break;
+				case P_features.Warlord7:
+				    var lop;lop="Ork Warboss discovered on "+string(name)+" "+scr_roman(planet_num)+"."debugl(lop);
+				    scr_alert("red","feature",lop,x,y);scr_event_log("red",lop);
+					break;
+		
+			
+			}
+		}
+	}
+}
+		
 
 	// self: obj_star
```

Comment:
shouldn't we delete this whole commented out section?

Reply:
yeah will do